JS-Algorithms/challenges.js

```diff
@@ -70,7 +83,17 @@ The last 3 digits for the sum of powers from 1 to 10 is "317"
 ***** */
 
 const ownPower = (number, lastDigits) => {
-  // YOUR CODE HERE...
+  let total = 0;
+  //Create an array with elements and multiply by each own powers
+  const range = Array.from({ length: number }, (item, index) =>
+    Math.pow(index + 1, index + 1)
+  );
+  //Sum all elements
+  range.forEach((item) => (total += item));
```

Comment:
I understand that we need two paremeters  in the callback function, but since item is not used we get this message

![image](https://user-images.githubusercontent.com/66694908/233888423-76a98ecd-eeb0-4623-abbe-3c9dbb9af26a.png)

To avoid lintern errors or misunderstanding we could use '_'